### DISCUSSION TITLE
Error on getting login request

### DISCUSSION THREAD

**Discussion by dselyuzhitskiy** [ory/hydra#2375] [Q&A] [answered]
Hello!

Sometimes we get an error while making query to Hydra, getting login request. 

> "error": "[GET /oauth2/auth/requests/login][409] getLoginRequestConflict  \u0026{Debug: Error:0xc000fbc5b0 ErrorDescription:Unable to process the requested resource because of conflict in the current state StatusCode:0}",

I found that its because challenge, that used in this query, already used ( WasUsed flag in DB). Also i found that this flag is set while making request to /oauth/auth with login_verifier ( next step after log in in our login application ). But i have no idea why this error happen and what situation could be, maybe anyone had same issue?

**Comment by Benehiko** [comment, marked as answer]
Hi @dselyuzhitskiy,

This issue looks similar to what you are experiencing https://github.com/ory/hydra/issues/1569

**Reply by dselyuzhitskiy** [reply]
Thanks! This is exact our problem
